example/android/build.gradle

```diff
@@ -3,9 +3,9 @@
 buildscript {
     ext {
         buildToolsVersion = "33.0.0"
-        minSdkVersion = 24
-        compileSdkVersion = 33
-        targetSdkVersion = 33
+        minSdkVersion = 22
```

Comment:
The previous version was 21 (https://github.com/underdog-tech/react-native-pinwheel/blob/4ee4b2e07b215ffe84d73581bd883f5575556679/example/android/build.gradle) Does this matter?

Reply:
we can't really go down to 22 without an Android release. 21 and 22 are both "Lollipop" releases, so I _think_ it should be okay.